resolwe/elastic/__init__.py

```diff
@@ -0,0 +1,35 @@
+""".. Ignore pydocstyle D400.
+
+=======
+Elastic
+=======
+
+Framework for advanced indexing of Django models with ElasticSearch.
+
+To register index processor, create `elastic_indexes.py` file int your
+app and put subclass of :class:`~resolwe.elastic.indices.BaseIndex` in
+it. It will automatically register and index all objects specified in
+it.
+
+For building the index for the first time or manually updating it, run::
+
+    python manage.py elastic_index
+
+.. automodule:: resolwe.elastic.indices
+.. automodule:: resolwe.elastic.viewsets
+.. automodule:: resolwe.elastic.builder
+.. automodule:: resolwe.elastic.utils
+.. automodule:: resolwe.elastic.management.commands
+
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from elasticsearch_dsl.connections import connections
+
+from django.conf import settings
+
+elasticsearch_host = getattr(settings, 'ELASTICSEARCH_HOST', 'localhost')  # pylint: disable=invalid-name
+elasticsearch_port = getattr(settings, 'ELASTICSEARCH_PORT', 9200)  # pylint: disable=invalid-name
+connections.create_connection(hosts=['{}:{}'.format(elasticsearch_host, elasticsearch_port)])
```

Comment:
Thanks for spotting this, Ive forgot to revert the changes when performing some tests.